Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -669,14 +669,17 @@ static void BPWritten(const BPCmd& bp)
   // --------------------------------------------
   case BPMEM_TX_SETIMAGE0:
   case BPMEM_TX_SETIMAGE0_4:
+  case BPMEM_TX_SETIMAGE3:
+  case BPMEM_TX_SETIMAGE3_4:
+    TextureCacheBase::InvalidateAllBindPoints();
+    return;
   case BPMEM_TX_SETIMAGE1:
   case BPMEM_TX_SETIMAGE1_4:
   case BPMEM_TX_SETIMAGE2:
   case BPMEM_TX_SETIMAGE2_4:
-  case BPMEM_TX_SETIMAGE3:
-  case BPMEM_TX_SETIMAGE3_4:
-    TextureCacheBase::InvalidateAllBindPoints();
+    TmemBase::ConfigurationChanged(bp.address, bp.newvalue);
```

Comment:
It feels odd that 0 and 3 do something differently to 1 and 2. Maybe it would be a good idea to leave a few comments on _why_ they're different?